content/en/docs/concepts/cluster-administration/events.md

```diff
@@ -0,0 +1,34 @@
+---
+reviewers:
+- piosz
+- x13n
+content_template: templates/concept
+title: Events in Kubernetes
+---
+
+{{% capture overview %}}
+
+Kubernetes events are objects that provide insight into what is happening
+inside a cluster, such as what decisions were made by scheduler or why some
+pods were evicted from the node. 
+{{% /capture %}}
+
+
+{{% capture body %}}
+
+Since events are API objects, they are stored in the {{< glossary_tooltip text="apiserver" term_id="kube-apiserver" >}} on master. To
```

Comment:
```suggestion
Since Events are API objects, they are stored in the {{< glossary_tooltip text="apiserver" term_id="kube-apiserver" >}} on the control plane. To
```

Reply:
done